languages-team.json

```diff
@@ -7,8 +7,13 @@
         "at any time"
     ],
     "enabledManagers": [
+        "github-actions",
         "regex"
     ],
+    "addLabels": ["dependencies"],
+    "github-actions": {
+        "fileMatch": ["^(github-action)(\\/[^/]+)?\\/action\\.ya?ml$"]
+    },
```

Comment:
I would not extend [the defaults](https://docs.renovatebot.com/modules/manager/github-actions/#default-config) here. The feature is undocumented and [it is not used in any repositories](https://github.com/search?q=org%3ASonarSource+path%3Agithub-action%2F**%2Faction.y*ml&type=code).